{
    "Computer":  "VSDC1",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1759820571313)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Windows cannot load the extensible counter DLL \"C:\\Windows\\System32\\iasperf.dll\" (Win32 error code The specified module could not be found.).",
                       "Id":  1023,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  83013
                   },
                   {
                       "TimeCreated":  "\/Date(1759814611400)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Error calling a routine on a Shadow Copy Provider {74600e39-7dc5-4567-a03b-f091d6c7b092}. Routine details BeginPrepareSnapshot({8c8f9bf4-e2c1-469f-af8d-b167b1cb73a9},{988a335e-46ec-4510-9697-0acb483e1500},\\\\?\\Volume{c6def331-2795-4a00-bf35-64cc5b01fbec}\\) [hr = 0x8000ffff, Catastrophic failure\r",
                       "Id":  12293,
                       "ErrorCode":  "0x8000ffff",
                       "UpdateTitle":  null,
                       "EventRecordID":  82938
                   },
                   {
                       "TimeCreated":  "\/Date(1759813347747)\/",
                       "Source":  "Volsnap",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The shadow copies of volume \\\\?\\Volume{b1bc169e-a33a-11f0-ab88-00155d611102} were aborted during detection because a critical control file could not be opened.",
                       "Id":  27,
                       "ErrorCode":  "0x0001",
                       "UpdateTitle":  null,
                       "EventRecordID":  198622
                   },
                   {
                       "TimeCreated":  "\/Date(1759785986573)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{45f4bc6b-42e6-4fce-aaaf-dfafa08cec5b}\" of \"\\Microsoft\\Windows\\Servicing\\StartComponentCleanup\"  task for user \"NT AUTHORITY\\SYSTEM\" . Additional Data: Error Value: 2147942405.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  160382
                   },
                   {
                       "TimeCreated":  "\/Date(1759785986573)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\Servicing\\StartComponentCleanup\" , instance \"{45f4bc6b-42e6-4fce-aaaf-dfafa08cec5b}\" , action \"Component Based Servicing Session\" . Additional Data: Error Value: 2147942405.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  160381
                   },
                   {
                       "TimeCreated":  "\/Date(1759741321800)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\Workplace Join\\Automatic-Device-Join\" task for user \"(default user)\". Additional Data: Error Value: 2147943568.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  158583
                   },
                   {
                       "TimeCreated":  "\/Date(1759741292624)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Access to performance data was denied to user \"SYSTEM\" (value from GetUserName() for the running thread) as attempted from module \"C:\\Program Files (x86)\\NinjaOne\\NinjaRMMAgent.exe\" (value from GetModuleFileName() for the binary that issued the query).",
                       "Id":  1000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  82031
                   },
                   {
                       "TimeCreated":  "\/Date(1759741280060)\/",
                       "Source":  "Microsoft-Windows-DistributedCOM",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "DCOM got error \"1115\" attempting to start the service TrustedInstaller with arguments \"Unavailable\" in order to run the server:\r",
                       "Id":  10005,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  196785
                   }
               ],
    "LastPoll":  "10/07 01:27"
}